_includes/sidenav.html

```diff
@@ -24,16 +23,17 @@
 
 <nav class="nav-secondary expandable {{ additional_classes }}">
     <h1 class="u-visually-hidden">
```

Comment:
_Adds it to the backlog_

Signing it off with "Thanks Scott!!!" :stuck_out_tongue_closed_eyes: